geometry_msgs/Twist data 	# data about what movements robot should follow
---
bool ack 			# acknowledged informations 
